apps/nextjs-website/src/app/layout.tsx

```diff
@@ -70,6 +71,7 @@ export default async function RootLayout({
           )}
           <AuthProvider>
             <SiteHeader products={products} />
+            <WebinarHeaderBanner slug={products[0].slug}></WebinarHeaderBanner>
```

Comment:
this component should only be visible in home so it should be moved to `apps/nextjs-website/src/app/page.tsx`